sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -82,7 +82,7 @@ export const studentRoutes: Array<RouteRecordRaw> = [
         },
       },
       {
-        path: "application-form/:id?",
+        path: "application-form/:id?/:selectedForm",
```

Comment:
as a recommendation , can we add the path in `https://github.com/bcgov/SIMS/blob/main/sources/packages/web/src/types/AppRoutes.ts` and call here